main.go

```diff
@@ -49,7 +49,11 @@ func main() {
 	flag.StringVar(&prometheusNamespace, "prometheus-namespace", "",
 		"Prometheus namespace for exported metrics.")
 	flag.UintVar(&extPort, "extport", 443,
-		"Nitriding's VSOCK-facing HTTPS port.  Must match port forwarding rules on EC2 host.")
+		"Nitriding's HTTPS port.  Must match port forwarding rules on EC2 host.")
```

Comment:
The port is still "facing" the VSOCK interface, no?

Reply:
if `-vsock-ext` is not enabled, then it would be considered "ipv4/ipv6-facing" right?

Reply:
Ah, yes, that's right. By "VSOCK-facing" I meant "reachable to the VSOCK interface", even if it means that packets first go through the kernel's network stack. Anyway, let's get rid of the "VSOCK-facing" because it probably causes less confusion.